INSTALL.md

```diff
@@ -1,58 +1,63 @@
 Building QGIS from source - step by step
 
-<!-- Table of contents generated with https://freelance-tech-writer.github.io/table-of-contents-generator/index.html -->
+<!-- TOC start (generated with https://github.com/derlin/bitdowntoc) -->
```

Comment:
This URL points to a github repo where I don't see direct and easy way to generate the TOC. We don't want to do coding just for a TOC.
Maybe should we point to his website https://bitdowntoc.derlin.ch/ or keep the older URL, because you simply have to copy paste your file, and DONE!